substrate/client/network/src/peer_store.rs

```diff
@@ -222,7 +222,7 @@ impl PeerStoreInner {
 		if peer_info.reputation < BANNED_THRESHOLD {
 			self.protocols.iter().for_each(|handle| handle.disconnect_peer(peer_id));
 
-			log::trace!(
+			log::error!(
```

Comment:
I would prefer warning instead of error.

Reply:
Agree! Done.